other/materials_designer/create_cluster_sphere.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    
</pre><p>Remove newline</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It's a formatter working. Previously we didn't run it. </p>